piano-hero/core/src/main/java/com/novoda/pianohero/KeyListener.java

```diff
@@ -0,0 +1,8 @@
+package com.novoda.pianohero;
+
+interface KeyListener {
```

Comment:
i'm not sure if this should be in core. it means _key_ as in _keyboard_ right, what about a voice interface? that would not have a key, therefore shouldn't this stay out of core?